installation_tips/full_spikeinterface_environement_rolling_updates.yml

```
@@ -0,0 +1,35 @@
+name: si_env
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.11
+  - pip
+  - numpy
+  - scipy
+  - joblib
+  - tqdm
+  - matplotlib
+  - h5py
+  - pandas
+  - xarray
+  - hdbscan
+  - scikit-learn
+  - networkx
+  - pybind11
+  - loky
+  - numba
+  - jupyter
+  - pyqt=5
+  - pyqtgraph
+  - ipywidgets
+  - ipympl
+  - pip:
+    - ephyviewer
+    - docker
```

```suggestion
    - docker
```

Is docker really necessary... Seems heavy for this.

---

The docker python is quite small no ?
Sometimes this is good to have when teaching.

---

You're the one that does way more teaching. I just wasn't sure how many people set up docker. But if it works for your flow I think it is fine to keep.

---

I leave it. Lets see.